### PR TITLE
Hotfixes

### PR DESCRIPTION
**Pull request by Mster115**
- [ ] No discrepancies across browsers. Eg. chrome vs safari
- [ ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ ] CSS (https://jigsaw.w3.org/css-validator/)
- [ ] Google sign-in is operational
- [ ] Accessibility functions
- [ ] Pages can scale without distorting page
- [ ] No dead links
- [ ] Navbar is consistent across the site
- [ ] Pages are screen-reader accessible
- [ ] Contrast meets standards for accessibility
- [ ] Pages are keyboard accessible
- [ ] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

**Comment by st5626** [comment]
Things look great! Good job!!

File: client/src/components/body/lab5/about.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const About = (lab) => {
                 In this lab, you will learn about why it is important to create software
                 that is accessible to users who face cognitive impairments.
                 You will learn about using clear descriptive headings, handling time driven notifications, and
-                creating informative form responses. Afterwards, you will watch related videos and take a quiz
+                creating informative form responses. Afterwards, you will view related media to reinforce the topic and take a quiz
```
Comment (heathermoses) [inline client/src/components/body/lab5/about.js:18]: This is super nitpicky, but I think there should be a comma after "topic"

Comment (heathermoses) [reply client/src/components/body/lab5/about.js:18]: Oh wait nevermind! Just read this more carefully, my bad

File: client/src/components/body/landingpage/citation/professorInformation.js [hunk 2]
```diff
@@ -7,14 +7,16 @@ const professorInformation = [
...
+        socials: ["https://danielkrutz.github.io/"],
+        network: "sharethis"
     },
     {
         profile_image: samuel_malachowsky_image,
         name: "Samuel Malachowsky",
         title: "Co-PI",
         bio: "",
         socials: ["https://www.se.rit.edu/~samvse/"]
+
```
Comment (heathermoses) [inline client/src/components/body/landingpage/citation/professorInformation.js:19]: Going off of the addition to krutz's info, should there be a network field for malachowsky?

File: client/src/components/exercise/lab1/components/Countdown.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ class Countdown extends Component {
 			<Fragment>
 				<CongratulationMessage message={message} />
 
-				<div className="game__countdown">{time}</div>
+				<div className="Exercise__countdown">{time}</div>
```
Comment (heathermoses) [inline client/src/components/exercise/lab1/components/Countdown.js:15]: Idk if this makes a difference, but could "Exercise__countdown" be lowercase? Just to follow the convention before of "game__countdown"

File: client/src/components/exercise/lab1/pages/Fourth.js [hunk 4]
```diff
@@ -95,13 +95,13 @@ class Fourth extends Component {
 		return (
 			<div className="playthrough">
 				<div className="playthrough__content">
-					<p className="playthrough__sentence">Here are your results from all three games:</p>
+					<p className="playthrough__sentence">Here are your results from all three Exercises:</p>
```
Comment (heathermoses) [inline client/src/components/exercise/lab1/pages/Fourth.js:98]: Maybe Exercises can be lowercase?

**Review by heathermoses — APPROVED** [review]
Everything looks good to me! Great job on this!

**Pull request merged**